website/static/js/fangorn.js

```diff
@@ -1157,11 +1157,11 @@ function _fangornUploadMethod(item) {
     return configOption || 'PUT';
 }
 
-function gotoFileEvent (item) {
+function gotoFileEvent (item, toUrl) {
```

Comment:
Don't feel the need to change this, but if you'd wanted to you could have checked to see if toUrl was defined and, if not, set it to '/'. That would have kept the interface the same for existing usages while still allowing you to add the new functionality.